easybuild/easyblocks/a/amber.py

```diff
@@ -141,9 +143,17 @@ def install_step(self):
                 self.mpi_option = '-mpi'
 
         common_configopts = [self.cfg['configopts'], '--no-updates', '-noX11']
-        if self.name == 'Amber':
+        if self.name == 'Amber' and self.cfg['static']:
             common_configopts.append('-static')
 
+        envval = lambda ev: os.getenv(ev) or ''
```

Comment:
@valtandor You don't need this, you can use `os.getenv('AMBERBUILDFLAGS', '')` instead